File: app/src/main/java/com/dailyvery/apps/imhome/LoaderSplashScreen.java
package com.dailyvery.apps.imhome;

import android.content.Intent;
import android.os.Handler;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class LoaderSplashScreen extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_loader_splash_screen);

        Handler handler = new Handler();
        handler.postDelayed(new Runnable() {
            public void run() {
                Intent intent = new Intent(LoaderSplashScreen.this, MainActivity.class);
                startActivity(intent);
                finish();
            }
        }, 600);


    }
}

File: app/src/main/java/com/dailyvery/apps/imhome/LoaderSplashScreen.kt
package com.dailyvery.apps.imhome

import android.content.Intent
import android.os.Handler
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class LoaderSplashScreen : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_loader_splash_screen)

        val handler = Handler()
        handler.postDelayed({
            val intent = Intent(this@LoaderSplashScreen, MainActivity::class.java)
            startActivity(intent)
            finish()
        }, 600)


    }
}
